CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 
+### Breaking changes:
+- Added multiprocess support for the Spatial functional test framework.
+- Modified the functional test startup flow in engine and GDK to accomodate both single and multiprocess tests.
+- Modified module loading process in the engine for GDKTestGymsFunctionalTest to ensure module are correctly loaded in newly spawned processes.
```

Comment:
Currently, the testing framework is classified mostly as "internal", so I would move these things there.
Also, I think you can leave out the module loading changes, as I think they are just a strict relaxation of dependencies?

Reply:
Sure, will make that change now